README.md

```diff
@@ -35,13 +35,14 @@ $ pip-compile
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt setup.py
+#    pip-compile --output-file requirements.txt requirements.in
```

Comment:
This was an example to use `pip-compile` straight with a setup.py file, without a `requirements.in`, so the example should be kept as before.